### PR TITLE
Add ability to disable automatic starting of the generated systemd units

### PR DESCRIPTION
**Pull request by josephschorr**

### CODE REVIEW

File: container_cloud_config/templates/dockersystemd.yaml [hunk 1]
```diff
@@ -43,8 +47,8 @@
     {% else -%}
     Restart={{ restart_policy }}
     {%- endif %}
-    TimeoutStartSec=600
-    TimeoutStopSec=2000
+    TimeoutStartSec={{ timeout_start_sec|default(600) }}
```
Comment (bison) [inline container_cloud_config/templates/dockersystemd.yaml:50]: These timeouts are way higher than the defaults for regular units and infinitely higher than the default for oneshots. Can we make this optional? I.e. only include this bit if they are defined?

Comment (bison) [reply container_cloud_config/templates/dockersystemd.yaml:50]: Oh, I missed that they were already really high, heh.

**Review by bison — APPROVED** [review]
LGTM

File: container_cloud_config/templates/dockersystemd.yaml (hunk 1 above)
Comment (josephschorr) [reply container_cloud_config/templates/dockersystemd.yaml:50]: Yeah, this will allow everyone to override them

**Pull request merged**